include/xtensor/xbroadcast.hpp

```diff
@@ -364,6 +368,15 @@ namespace xt
         // Could check if (broadcastable(shape, m_shape)
         return m_e.stepper_end(shape, l);
     }
+
+    template <class CT, class X>
+    template <class E, class XCT, class>
+    inline void xbroadcast<CT, X>::assign_to(xexpression<E>& e) const
+    {
+        auto& ed = e.derived_cast();
+        ed.resize(m_shape);
+        std::fill(ed.begin(), ed.end(), m_e());
```

Comment:
that does not appear to be correct, because `m_e` can more than one-dimensional in xbroadcast.